src/app/app.module.ts

```diff
@@ -11,29 +11,18 @@ import { ClientService } from './smartonfhir/client.service';
 import { RecordDataComponent } from './record-data/record-data.component';
 import { ResultDetailsComponent } from './result-details/result-details.component';
 import { TrialCardComponent } from './trial-card/trial-card.component';
+import Client from 'fhirclient/lib/Client';
 
 const fhirInitializeFn = (fhirService: ClientService) => {
   // Grab the client during bootstrap - this prevents the flash of a partially
   // loaded client if SMART on FHIR needs to do an OAuth authentication prior to
   // continuing to bootstrap the Angular app
-  return () => fhirService.getClient();
+  return (): Promise<Client> => fhirService.getClient();
 };
 
 @NgModule({
-  declarations: [
-    AppComponent,
-    RecordDataComponent,
-    ResultDetailsComponent,
-    TrialCardComponent
-  ],
-  imports: [
-    BrowserModule,
-    AppRoutingModule,
-    HttpClientModule,
-    FormsModule,
-    NgxSpinnerModule,
-    BrowserAnimationsModule
-  ],
+  declarations: [AppComponent, RecordDataComponent, ResultDetailsComponent, TrialCardComponent],
+  imports: [BrowserModule, AppRoutingModule, HttpClientModule, FormsModule, NgxSpinnerModule, BrowserAnimationsModule],
```

Comment:
Weird that these got compressed. Might want to ignore this change

Reply:
Not really, it's an artifact of me extending the line length to be 120 instead of the default 80: the items technically fit within the longer lines. Of course, the reason I extended the line length is that 80 compresses a *lot* of stuff we probably don't want to compress.

This is another example of where we'd have to selectively disable prettier to silence it because I agree, it's less readable, but it is technically (and just barely) correct: the imports line is exactly 120 characters wide.

Reply:
I think this should be disabled